flexget/plugins/sites/freshon.py

```diff
@@ -0,0 +1,188 @@
+                            if page_int > max_page_number:
+                                max_page_number = page_int
+                else:
+                    max_page_number = 1
+
+                # limit to 10 pages
+                if max_page_number > 10:
+                    max_page_number = 10
+
+                # pages are 0 based
+                nextpage = 0
+                while (nextpage < max_page_number):
+                    if (nextpage > 0):
+                        newurl = url + '&page=' + str(nextpage)
+                        log.debug('-----> NEXT PAGE : %s' % newurl)
+                        f1 = task.requests.get(newurl).content
+                        soup = get_soup(f1)
+                    for res in soup.findAll('tr', {'class': re.compile('torrent_[0-9]*')}):
+                        entry = Entry()
+                        # skip if nuked
+                        if res.find('img', alt='Nuked') != None:
+                            continue
+
+                        link = res.find('a', attrs={'href': re.compile('dl-torrent')})
+
+                        entry['title'] = res.find('a', {'class': 'torrent_name_link'})['title']
+
+                        details_url = res.find('a', {'class': 'torrent_name_link'})['href']
+                        id = int((re.match('.*?([0-9]+)$', details_url).group(1)).strip())
+                        str_url = (base_url, dl_fragment)
+                        entry['url'] = '/'.join(str_url) % (str(id), config['passkey'])
+
+                        log.debug('Title: %s | DL LINK: %s' % (entry['title'], entry['url']))
+
+                        seeds = int(res.find('td', {'class': 'table_seeders'}).find('span').text.strip())
+                        leechers = int(res.find('td', {'class': 'table_leechers'}).find('a').text.strip())
+                        entry['torrent_seeds'] = seeds
+                        entry['torrent_leeches'] = leechers
+
+                        size = res.find('td', attrs={'class': re.compile('table_size')}).text
+                        size = re.search('(\d+(?:[.,]\d+)*)\s?([KMG]B)', size)
+
+                        if size:
+                            if size.group(2) == 'GB':
+                                entry['content_size'] = int(float(size.group(1)) * 1000 ** 3 / 1024 ** 2)
+                            elif size.group(2) == 'MB':
+                                entry['content_size'] = int(float(size.group(1)) * 1000 ** 2 / 1024 ** 2)
+                            elif size.group(2) == 'KB':
+                                entry['content_size'] = int(float(size.group(1)) * 1000 / 1024 ** 2)
+                            else:
+                                entry['content_size'] = int(float(size.group(1)) / 1024 ** 2)
+
+                        if (int(entry['torrent_seeds']) > 0):
+                            entries.add(entry)
+                        else:
+                            log.debug('0 SEEDS, not adding entry')
```

Comment:
I'm a little puzzled by this and I assume it was always there. That should not be up to the search plugin decide. We already have https://flexget.com/Plugins/torrent_alive if people don't want dead torrents.